src/main/java/frc/robot/subsystems/DriveBaseSubsystem.java

```diff
@@ -54,18 +59,26 @@ public void periodic() {
 
   // Normal Arcade Drive
   public void arcadeDrive() {
-    m_differentialDrive.arcadeDrive( m_driverJoystick.getRawAxis(Constants.kDBLeftJoystickAxisY), m_driverJoystick.getRawAxis(Constants.kDBRightJoystickAxisY));
+    m_differentialDrive.arcadeDrive( m_driverJoystick.getRawAxis(Constants.kDBLeftJoystickAxisY), 
+                                      m_driverJoystick.getRawAxis(Constants.kDBRightJoystickAxisY));
 
   }
 
+  // Arcade Drive where you can only move forwards and backwards for testing
+  public void arcadeDrive(double rotation) {
+    m_differentialDrive.arcadeDrive(m_driverJoystick.getRawAxis(Constants.kDBLeftJoystickAxisY), rotation);
+  }
+
   // tank drive, not used but good to have
   public void tankDrive() {
-    m_differentialDrive.tankDrive(m_driverJoystick.getRawAxis(Constants.kDBLeftJoystickAxisY), m_driverJoystick.getRawAxis(Constants.kDBRightJoystickAxisY));
+    m_differentialDrive.tankDrive(m_driverJoystick.getRawAxis(Constants.kDBLeftJoystickAxisY), 
+                                  m_driverJoystick.getRawAxis(Constants.kDBRightJoystickAxisY));
   }
 
-  // Arcade Drive where you can only move forwards and backwards for testing
-  public void arcadeDrive(double rotation) {
-    m_differentialDrive.arcadeDrive(m_driverJoystick.getRawAxis(Constants.kDBLeftJoystickAxisY), rotation);
+  public void arcadeDriveVolts(double leftVolts, double rightVolts) {
```

Comment:
This wouldn't be arcade drive anymore. Maybe name it something like setVoltsForAuton